libqtile/scratchpad.py

```diff
@@ -285,6 +287,7 @@ def on_float_change(self, *args, **kwargs):
         for name, dd in self.dropdowns.items():
             if not dd.window.floating:
                 if dd.window.group is not self:
+                    dd.window.qtile.unpin(dd.window.window.wid)
```

Comment:
What's this for?

Reply:
I decided to "pin" Scratchpads to the top; that (and the other change in that file) basically from the other z-PR, where it was because you at some point mentioned that scratchpads didn't work correctly.

I should test whether that line is even necessary!

Reply:
I removed the changes in this file, since scratchpads seem to work fine without them. Might have to think about this, though, since I didn't expect that.